plugins/chat/spec/serializer/chat/message_interaction_spec.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+RSpec.describe Chat::MessageInteractionSerializer do
+  subject(:serializer) do
+    interaction = Chat::CreateMessageInteraction.call(params:, guardian: user.guardian).interaction
+    described_class.new(interaction, scope: Guardian.new(user), root: false)
+  end
+
+  fab!(:user)
+
+  let(:message) do
+    Fabricate(
+      :chat_message,
+      chat_channel: channel,
+      user: Discourse.system_user,
+      blocks: [
+        {
+          type: "actions",
+          elements: [
+            { type: "button", text: { type: "plain_text", text: "Like" }, action_id: "like" },
+          ],
+        },
+      ],
+    )
+  end
+  let(:message_id) { message.id }
+  let(:params) { { message_id: message_id, action_id: "like" } }
```

Comment:
Maybe a nitpick: we can write `{ message_id:, action_id: "like" }` :slightly_smiling_face: